UPGRADE.md

```diff
@@ -2,6 +2,11 @@
 
 ## Version 4.* to 5.0
 
+### database.sql files
+
+Support for `database.sql` files has been dropped. Use DCA definitions and/or Doctrine DBAL schema
```

Comment:
I think this should say _use Doctrine entities instead_?

Reply:
No? Use whatever you want of Doctrine :)

Reply:
yeah but you 99.9% don't want a **DBAL schema listener**. You would either write a DCA file or a Doctrine **ORM** entity?

Reply:
also btw, what we use is a ORM schema listener if I remember correctly 😅 not sure there is a DBAL listener at all.

Reply:
Of course https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/events.html#schema-events. Anyway, leaving this as is.